src/xlsx/functions/charts.ts

```diff
@@ -211,6 +211,72 @@ function insertTextProperties(
   `;
 }
 
+function extractTrendline(
+  trend: ExcelChartDataset["trend"],
+  dataSetColor: XlsxHexColor
+): XMLString {
+  if (!trend) return escapeXml/*xml*/ ``;
+  const { type, order, window } = trend;
+  const trendLineNodes: XMLString[] = [];
+  switch (type) {
+    case "poly":
+      if (order && order > 1) {
+        trendLineNodes.push(escapeXml/*xml*/ `<c:trendlineType val="poly" />`);
+        trendLineNodes.push(escapeXml/*xml*/ `<c:order val="${order}" />`);
+      } else {
+        trendLineNodes.push(escapeXml/*xml*/ `<c:trendlineType val="linear" />`);
+      }
+      break;
+    case "movingAvg":
+      trendLineNodes.push(escapeXml/*xml*/ `<c:trendlineType val="movingAvg" />`);
+      if (window) {
+        trendLineNodes.push(escapeXml/*xml*/ `<c:period val="${window}" />`);
+      }
+      break;
+    default:
+      trendLineNodes.push(escapeXml/*xml*/ `<c:trendlineType val="${type}" />`);
+      break;
+  }
+  return escapeXml/*xml*/ `
+    <c:trendline>
+      ${extractTrendlineCommonAttributes(trend, dataSetColor)}
+      ${joinXmlNodes(trendLineNodes)}
+    </c:trendline>
+  `;
+}
+
+function extractTrendlineCommonAttributes(
+  trend: ExcelChartDataset["trend"],
+  dataSetColor: XlsxHexColor
+): XMLString {
+  if (!trend) {
+    return escapeXml/*xml*/ ``;
+  }
+  const attrs: XMLAttributes = [
+    [
+      "val",
+      trend.color ? toXlsxHexColor(trend.color).slice(0, 6) : getTrendlineColor(dataSetColor),
```

Comment:
`toXlsxHexColor(trend.color).slice(0, 6)` is wrong, xlsx color have the alpha as the first two digits (AARRGGBB). You will slice the blue channel instead of the alpha channel here.